zebedee-cms/src/main/java/com/github/onsdigital/zebedee/keyring/KeyringImpl.java

```diff
@@ -102,12 +103,41 @@ public SecretKey get(User user, Collection collection) throws KeyringException {
 
     @Override
     public void remove(User user, Collection collection) throws KeyringException {
-        // TODO
+        validGetParams(user, collection);
```

Comment:
I think it's time to refactor this method as it's used in more than just the Get key method now.

Can you split `validGetParams` into 2 separate methods:
- `validateUser(User user)` and 
- `validateCollection(Collection collection)` 

You can cut and paste logic from the existing method. Then update the each method to use whichever is required.